### PR TITLE
Added readme.md to config component.

### PR DESCRIPTION
**Pull request by Rubemlrm**
Good Night,

I've added a README.MD example into config component. Basically i added a small component description, and added a little more information for the given component usage.

Besides this i've removed the reference to this component from other packages sections.

I've tried to maintain the original README.MD template and provide only extra information relative to this component. 

If this is all right , i will try do the same for the rest of components too.

### CODE REVIEW

**Comment by Rubemlrm** [comment]
@mattstauffer hi there . sorry for bothering about this pull request. I would like to get some opinion relative to this , so i can do the same for the other components =).

P.S: Hope its all fine with you and yours

File: components/config/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# <img src="torch-logo.png" alt="Torch Logo" align="right">Torch - Using Laravel's Illuminate Components Independently
+
+Torch is a project to provide instructions and examples for using Illuminate components as standalone components in non-Laravel applications. The current `master` branch shows how to use Illuminate's `5.1` components.
+
+**Note**: If you are working with an older project, you might have more success using the [4.2 components](https://github.com/mattstauffer/torch/tree/4.2).
+
+## Config Component
+
+The config component tries to demonstrate how we can use the [Laravel Config](https://laravel.com/docs/5.4/configuration/) package to store our configs in a array and display the same.
```
Comment (m1guelpf) [inline components/config/README.md]: Maybe update the documentation link to the 5.5 version?

Comment (Rubemlrm) [reply components/config/README.md]: I thinks this components are related to 5.4 only . I need to check if they work with 5.5 too.

Comment (mattstauffer) [reply components/config/README.md]: These components are 5.1. :)

**Comment by mattstauffer** [comment]
@Rubemlrm reviewing now--thanks!

File: components/config/README.md (hunk 1 above)
Comment (Rubemlrm) [reply components/config/README.md]: you are right i've mistake 4.4 with 5.4 shame on me 👎

**Comment by mattstauffer** [comment]
@Rubemlrm Thanks for this! I simplified it a bit and you can see it here: 

https://github.com/mattstauffer/Torch/tree/master/components/config

**Comment by Rubemlrm** [comment]
@mattstauffer thats way cleaner and simple. I will do the same for the other components =)

**Pull request merged**